content-service/README.md

```diff
@@ -85,6 +84,22 @@ TODO: Pagination
 
 Get information about a beacon.
 
+```JS
+{
+  id: <string>,
+  short_url: <string>,
+  channel: <string>,
+  url: <string>,
+  call_to_action: <object>,
```

Comment:
No, it's a Map.  But I wanted to start changing from plurals to singulars.  It's a better convention and is generally how database table naming conventions go.  Currently the APIs are 'call_to_action' I think, while the database has a 'calls_to_action' field.  The database schema needs.. updating, it's a little off what the API became.